e2e-tests/playwright/e2e/plugins/github-issues/github-issues.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { expect, test } from "@playwright/test";
+import { UIhelper } from "../../../utils/ui-helper";
+import { Common } from "../../../utils/common";
+
+// Test GitHub Issues plugin functionality
+test.describe("GitHub Issues Plugin", () => {
+  const showcase = "Backstage Showcase";
+
+  let uiHelper: UIhelper;
+  let common: Common;
+
+  test.beforeEach(async ({ page }) => {
+    test.setTimeout(180000);
+    common = new Common(page);
+    await common.loginAsGithubUser();
+    uiHelper = new UIhelper(page);
+  });
+
+  test("Verify plugin in Extensions page", async ({ page }) => {
+    // Navigate to Extensions
+    await uiHelper.openSidebarButton("Administration");
+    await uiHelper.clickLink("Extensions");
+    await uiHelper.clickTab("Installed");
+
+    // Wait for page and search
+    await page.waitForSelector(
+      'table, [role="grid"], h2:has-text("Extensions")',
+      { timeout: 30000 },
+    );
+    await uiHelper.searchInputPlaceholder("github-issues");
+
+    // Verify plugin exists
+    const plugin = page
+      .locator(`text="backstage-community-plugin-github-issues"`)
```

Comment:
Reuse of Components
The code uses hardcoded strings like "github-issues" and "backstage-community-plugin-github-issues". It would be better to move these to reusable constants, for example:
`const PLUGIN_NAME = "backstage-community-plugin-github-issues";`

Reply:
done, declared variables definition in the beginning of the test